extra-radio/svxlink/autobuild/postinst

```diff
@@ -0,0 +1,12 @@
+/usr/bin/egrep -i "^svxlink" /etc/group >> /dev/zero
+if [ ! $? -eq 0 ]
+then
+	groupadd svxlink
+fi
+
+/usr/bin/egrep -i "^svxlink" /etc/passwd >> /dev/zero
```

Comment:
This is unsafe, what if there is `svxlink-daemon`? Could you please try and use `systemd-sysusers` here?